scripts/create_tables.sql

```diff
@@ -1 +1,29 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  date DATE,
+  time TEXT,
+  comment TEXT
+  doctor_id, INTEGER REFERENCES doctors(id)
+  patient_id, INTEGER REFERENCES patients(id)
+);
+CREATE INDEX ON appointments(doctor_id)
+CREATE INDEX ON appointments(patient_id)
```

Comment:
Good, don't need the CREATE INDEX though. We handle the REFERENCES to have the ID's linked.